tutorials/technology_demonstrations/arithmetic_expressions/arithmetic_expressions.ipynb

```diff
@@ -38,16 +38,16 @@
    },
```

Comment:
<p>Please verify that the result is printed as an output within the notebook. As you can see, the <code>result</code> variable is an execution Job object. First, you shall access the result itself <code>result.result()</code> and then print the relevant items (you can get the <code>parsed_counts</code> for example; See <a href="https://docs.classiq.io/latest/reference-manual/platform/executor/primitives/#sample" target="_blank">here</a> for more details. Also, it will be good to add some indicative message, something like:</p><p><code>print("The result of the arithmetic calculation:",...) </code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>